### PR TITLE
committing postgres_exercise

### PR DESCRIPTION
**Pull request by deepak-0912**

### CODE REVIEW

File: DBMS/exe1 [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Query 1:Find the Pincode with highest average sale price.
+
+
+answer:with temprarytable(pincode,highest_avg) as (select postcode,avg(price) as price from land_registry_price_paid_uk
```
Comment (Krishan410) [inline DBMS/exe1]: Please optimize..try to solve without subquery

**Comment by Krishan410** [comment]
remove csv file @deepak-0912

File: DBMS/schema-import [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Command for importing csv file into the table.
```
Comment (Krishan410) [inline DBMS/schema-import]: added total no of rows here

File: DBMS/exe1 (hunk 1 above)
Comment (deepak-0912) [reply DBMS/exe1]: ok sir

select postcode as pincode from land_registry_price_paid_uk group by pincode order by avg(price) desc  limit 1;

**Comment by Krishan410** [comment]
Good to Go. Merge in Master

**Pull request merged**